Controller/GluggiController.php

```diff
@@ -121,28 +121,42 @@ public function htmlTitle (GluggiConfig $config, $pageTitle)
      * @param GluggiConfig       $config
      * @param AssetHtmlGenerator $htmlGenerator
      * @param string             $type
+     * @param array              $addAssets
+     * @param array              $overrideAssets
+     *
      * @return Response
      * @throws \Becklyn\AssetsBundle\Exception\AssetsException
      */
-    public function layoutAssets (GluggiConfig $config, AssetHtmlGenerator $htmlGenerator, string $type) : Response
+    public function layoutAssets (GluggiConfig $config, AssetHtmlGenerator $htmlGenerator, string $type, array $addAssets, array $overrideAssets) : Response
     {
-        switch ($type)
+        $overrideAssets = $overrideAssets[$type] ?? [];
+
+        if (!empty($overrideAssets))
+        {
+            $assetPaths = $overrideAssets;
+        }
+        else
         {
-            case "js_head":
-                $assetPaths = $config->getJavaScriptHeadFiles();
-                break;
+            switch ($type)
+            {
+                case "js_head":
+                    $assetPaths = $config->getJavaScriptHeadFiles();
+                    break;
+
+                case "js":
+                    $assetPaths = $config->getJavaScriptFiles();
+                    break;
 
-            case "js":
-                $assetPaths = $config->getJavaScriptFiles();
-                break;
+                case "css":
+                    $assetPaths = $config->getCssFiles();
+                    break;
 
-            case "css":
-                $assetPaths = $config->getCssFiles();
-                break;
+                default:
+                    $assetPaths = [];
+                    break;
+            }
 
-            default:
-                $assetPaths = [];
-                break;
+            $assetPaths = \array_merge($assetPaths, $addAssets[$type] ?? null);
```

Comment:
`array_merge` expects an array as second parameter, so you must use `?? []` instead of `?? null`

Reply:
And that is why the tests are failing btw. So please also take a look at the tests and don't ignore the big red cross down there. 🙃